package.json

```diff
@@ -67,5 +69,6 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx,md,html,css,yml}": "npx prettier --write"
-  }
+  },
+  "dependencies": {}
```

Comment:
was an empty `dependencies` generated automatically?

Reply:
No I accidentally added and then removed a package